tool/src/org/antlr/v4/Tool.java

```diff
@@ -521,6 +521,19 @@ else if ( ruleAST==null ) {
 			// Make grammars depend on any tokenVocab options
 			if ( tokenVocabNode!=null ) {
 				String vocabName = tokenVocabNode.getText();
+				// Strip quote characters if any
+				int len = vocabName.length();
+				int firstChar = vocabName.charAt(0);
+				int lastChar = vocabName.charAt(len - 1);
+				if (len >= 2 && firstChar == '\'' && lastChar == '\'') {
+					vocabName = vocabName.substring(1, len-1);
+				}
```

Comment:
Removing the quotes could be done with less code:

`String vocabName = tokenVocabNode.getText().replaceAll("\\A'|'\\Z", "");`

It could be worthwhile to add a helper to misc.Utils. The Maven plugin also needs to strip quotes.